cscs-checks/apps/lammps/lammps_check.py

```diff
@@ -11,7 +11,10 @@
 
 class LAMMPSBaseCheck(rfm.RunOnlyRegressionTest):
     def __init__(self):
-        self.valid_prog_environs = ['builtin']
+        if self.current_system.name == 'pilatus':
+            self.valid_prog_environs = ['cpeGNU']
+        else:
+            self.valid_prog_environs = ['builtin']
```

Comment:
```suggestion
        else:
            self.valid_prog_environs = ['builtin']

```
Please add an extra new line after the if statements